dockerfiles/keycloak/kc_realm_user.sh

```diff
@@ -24,6 +21,12 @@ cat /scripts/che-realm.json.erb | \
                                 sed -e "s@<%= scope\.lookupvar('che::che_server_url') %>@${PROTOCOL}://che-${NAMESPACE}.${ROUTING_SUFFIX}@" \
                                 > /scripts/che-realm.json
 
+echo "Creating Admin user..."
+
+if [ $KEYCLOAK_USER ] && [ $KEYCLOAK_PASSWORD ]; then
```

Comment:
`! -z` ?

Reply:
Basically reused this https://github.com/jboss-dockerfiles/keycloak/blob/3.3.0.CR2-3/server/docker-entrypoint.sh

In fact, even a check isn't really required since ENVS are set in deploymentConfig.